docker/instant-seal-node.dockerfile

```diff
@@ -0,0 +1,42 @@
+# Requires to run from repository root and to copy the binary in the build folder
+FROM ubuntu:20.04 AS base
+LABEL maintainer="Frequency Team"
+LABEL description="Binary for Frequency parachain"
+
+RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates
+
+FROM ubuntu:20.04
```

Comment:
@demisx I recall that we tried once to switch to the much smaller alpine image, but it didn't work for some reason that I don't recall. Might be worth seeing what others are using as a base image.

Also I think we could bump up to the latest LTS if we are going to stick with Ubuntu